wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
I had to change the permissions of this file for docker-compose to run. I think you can do this and commit with git so other people don't have to do it: `chmod +x wait-for-it.sh`